### DISCUSSION TITLE
Best way to create a expected expense

### DISCUSSION THREAD

**Discussion by JeroenED** [firefly-iii/firefly-iii#7095] [Firefly III] [answered]
Hi,

Currently I have a few expenses coming that I already want to book into firefly-iii. Why? Mainly to be have a good view of what it's gonna be at the end of the month.

I have a possible day/time frame of when the expenses will occur and a estimate of the cost but no real data, yet.

So my question: what is the best way to create a one-time expense budget? Making a transaction in the future would make them dissappear in the crowd. So I need a clear distinction of "this transaction is budgetted"

**Comment by JC5** [comment, marked as answer]
One time? Just ALL CAPS them so they stand out?

**Reply by JeroenED** [reply]
Of course. Sometimes you have such simple solutions. Hope we'll get to know what happened last night! :D
